anycloud/cli/alan/anycloud.ln

```diff
@@ -138,7 +138,7 @@ fn getCleanError(error: string): string {
   return error
     .split("\n").join(" ")
     .split("").map(fn(value: string): string {
-      return value == '"' ? "'" : value;
+      return value == '"' ? '\"' : value;
```

Comment:
You may need to write this as `'\\"'` but I don't remember if we parse escape characters in single-quote strings or not.

Reply:
I've test it as is and it worked

Reply:
Then I guess we have a difference between single-quote and double-quote strings. Whether we want to keep that or not is something else, though.

Reply:
iirc we decided not to change the behavior in the current front-end, but to change the behavior with the change to the new front-end (since there will be some other breaking changes, ie in `@std`)

Reply:
For future reference, there wasn't actually a difference. :)